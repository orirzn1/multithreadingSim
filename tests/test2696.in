L14
S6
O7
N12

T0
I@0x00000000
LOAD $5, $7, 0x0
ADD $0, $6, $6
ADD $7, $5, $4
ADD $7, $3, $6
HALT $0

T1
I@0x00000000
ADD $7, $6, $3
ADDI $4, $4, 0x1
STORE $6, $0, $0
SUB $1, $4, $6
STORE $3, $7, 0x3
ADDI $1, $4, 0x1
SUB $7, $2, $6
HALT $0

T2
I@0x00000000
SUB $5, $4, $3
SUB $6, $0, $1
STORE $1, $5, $7
ADDI $1, $0, 0x4
ADD $0, $1, $6
ADDI $2, $6, 0x3
STORE $1, $5, $4
HALT $0

T3
I@0x00000000
LOAD $2, $1, $6
ADDI $1, $3, 0x1
STORE $5, $5, $0
ADD $4, $5, $7
ADD $6, $5, $2
HALT $0

T4
I@0x00000000
LOAD $2, $2, 0x0
ADDI $5, $5, 0x1
LOAD $2, $6, $3
LOAD $3, $3, 0x1
SUBI $5, $0, 0x3
ADD $2, $0, $3
HALT $0

T5
I@0x00000000
STORE $0, $0, $7
SUBI $7, $6, 0x1
LOAD $2, $3, 0x0
SUBI $3, $2, 0x2
ADDI $1, $3, 0x1
ADDI $7, $6, 0x1
LOAD $3, $6, 0x2
ADDI $5, $5, 0x3
SUB $7, $0, $3
STORE $2, $6, 0x3
HALT $0

T6
I@0x00000000
SUBI $5, $2, 0x3
ADDI $6, $0, 0x2
ADD $5, $6, $7
ADD $5, $4, $2
ADD $7, $7, $4
ADDI $2, $6, 0x3
ADDI $6, $1, 0x4
ADD $6, $2, $7
LOAD $4, $5, 0x2
SUBI $0, $7, 0x2
HALT $0

T7
I@0x00000000
ADD $0, $5, $1
ADDI $4, $0, 0x3
SUB $7, $3, $7
ADDI $3, $5, 0x3
STORE $0, $7, $3
LOAD $7, $3, $4
HALT $0

T8
I@0x00000000
ADDI $1, $2, 0x4
SUBI $4, $0, 0x1
LOAD $3, $1, $3
SUBI $2, $2, 0x4
HALT $0

T9
I@0x00000000
ADD $2, $5, $4
LOAD $5, $4, $7
ADDI $5, $2, 0x1
ADDI $4, $3, 0x2
HALT $0

T10
I@0x00000000
ADDI $0, $6, 0x3
ADDI $0, $2, 0x4
STORE $2, $5, 0x3
SUBI $1, $5, 0x1
ADDI $2, $4, 0x3
SUB $7, $3, $2
SUB $4, $6, $6
LOAD $2, $4, $0
STORE $7, $7, 0x3
HALT $0

T11
I@0x00000000
ADD $1, $6, $4
ADDI $4, $6, 0x4
ADDI $0, $1, 0x4
SUBI $3, $4, 0x3
SUB $0, $7, $6
SUB $0, $2, $5
LOAD $0, $0, $6
LOAD $3, $6, 0x1
HALT $0

D@0x0
0x4
0xA
0x3
0x6
0x7
0x9
0x1
0x8
0xA
