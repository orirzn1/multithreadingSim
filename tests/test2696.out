
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x4	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x2	R2 = 0x0	R3 = 0x0	R4 = 0x1	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x4	R1 = 0x4	R2 = 0x3	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x1	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFD	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0xFFFFFFFF	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x3	R6 = 0x0	R7 = 0x0
Register file thread id 6:
	R0 = 0xFFFFFFFE	R1 = 0x0	R2 = 0x5	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x5	R7 = 0x0
Register file thread id 7:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x3	R4 = 0x3	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 8:
	R0 = 0x0	R1 = 0x4	R2 = 0xFFFFFFFC	R3 = 0xA	R4 = 0xFFFFFFFF	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 9:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x2	R5 = 0x1	R6 = 0x0	R7 = 0x0
Register file thread id 10:
	R0 = 0x4	R1 = 0xFFFFFFFF	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFD
Register file thread id 11:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x4	R5 = 0x0	R6 = 0x0	R7 = 0x0
Blocked MT CPI for this program 3.663043

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x2	R2 = 0x0	R3 = 0x0	R4 = 0x1	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x4	R1 = 0x4	R2 = 0x3	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x1	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFD	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0xFFFFFFFF	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x3	R6 = 0x0	R7 = 0x0
Register file thread id 6:
	R0 = 0xFFFFFFFE	R1 = 0x0	R2 = 0x5	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x5	R7 = 0x0
Register file thread id 7:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x3	R4 = 0x3	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 8:
	R0 = 0x0	R1 = 0x4	R2 = 0xFFFFFFFC	R3 = 0x0	R4 = 0xFFFFFFFF	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 9:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x2	R5 = 0x1	R6 = 0x0	R7 = 0x0
Register file thread id 10:
	R0 = 0x4	R1 = 0xFFFFFFFF	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFD
Register file thread id 11:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x4	R5 = 0x0	R6 = 0x0	R7 = 0x0
Finegrained Multithreading CPI for this program 1.108696

